File: 5/solution.py
def gcd(a, b):
    if a == 0:
        return b
    return gcd(b % a, a)


def lcm(a, b):
    return a / gcd(a, b) * b


ans = 1
for i in range(2, 21):
    ans = lcm(ans, i)

print(ans)

File: 5/solution.cpp
#include <iostream>
using namespace std;
int gcd(int a, int b) {
    if (a == 0)
        return b;
    return gcd(b % a, a);
}
int lcm(int a, int b) { return a / gcd(a, b) * b; }
int main() {
    int ans = 1;
    for (int i = 2; i <= 20; i++)
        ans = lcm(ans, i);
    cout << ans << endl;
    return 0;
}
